docs/integrations/engines/databricks.md

```diff
@@ -38,6 +38,7 @@ Note: If using Databricks Connect please note the [requirements](https://docs.da
 | `databricks_connect_cluster_id`      | Databricks Connect Only: Databricks Connect cluster ID. Uses `http_path` if not set. Cannot be a Databricks SQL Warehouse.                                                                                                                                                                      | string |    N     |
 | `force_databricks_connect`           | When running locally, force the use of Databricks Connect for all model operations (so don't use SQL Connector for SQL models)                                                                                                                                                                  |  bool  |    N     |
 | `disable_databricks_connect`         | When running locally, disable the use of Databricks Connect for all model operations (so use SQL Connector for all models)                                                                                                                                                                      |  bool  |    N     |
+| `disable_spark_session`              | Do not use SparkSession if it is available (like when running in a notebook).                                                                                                                                                                                                                   |  bool  |    N     |
```

Comment:
Does this feature allow you to not use SparkSession even if it is available, or it allows you to not get warnings when it's not available?

Reply:
The former. So when running in a Databricks notebook you will always have a SparkSession available and this lets you say even though it is available you don't want to use it.